client/node/package.json

```diff
@@ -1,22 +1,30 @@
 {
-  "name": "@nitric/client",
-  "version": "1.0.0",
+  "name": "@veetwo/client",
+  "version": "0.1.0-alpha.2",
   "description": "nitric runtime client",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "engines": {
+    "node": ">=18"
+  },
   "scripts": {
     "bump": "standard-version",
-    "build": "tsc",
+    "build": "tsc && cp -r src/gen dist/gen",
```

Comment:
Why is this copy needed, doesn't the tsc already pick up the imported files and put them in the dist dir?